### ISSUE TITLE
ensembldb.ensembl.org NOT accessible

### ISSUE THREAD

**Issue by yanlolo** [Ensembl/ensembl-vep#1312]
## Describe the issue
I run the VEP command as below
```
vep -i $workspace/somatic/exome.merged.norm.pass_only.vcf --cache --dir /media/ssd/ngs-data-analysis/pmbio/bin/ensembl-vep/vep_cache/ --format vcf --vcf --plugin Downstream --plugin Wildtype --symbol --terms SO --flag_pick --transcript_version -o $workspace/somatic/exome.merged.norm.annotated.vcf 
```

but get the error
```
DBI connect('host=ensembldb.ensembl.org;port=3306','anonymous',...) failed: Can't connect to MySQL server on 'ensembldb.ensembl.org' (110) at /media/ssd/ngs-data-analysis/pmbio/bin/ensembl-vep/Bio/EnsEMBL/Registry.pm line 1771.
```

Then I also tried to ping ensembldb.ensembl.org and failed 
```
PING hl-ensembl-databases.ebi.ac.uk (193.62.192.245): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
Request timeout for icmp_seq 3

--- hl-ensembl-databases.ebi.ac.uk ping statistics ---
5 packets transmitted, 0 packets received, 100.0% packet loss
```

Could you please help? Thanks a lot

Regards,
Yiqing

**Comment by nakib103** [comment]
Hello @yanlolo ,

I tried re-producing your issue but failed - 
```
$ perl /hps/software/users/ensembl/repositories/snhossain/ensembl-vep/vep --force_overwrite --id rs699 --database --db_version 108
$
```

Were you able to run vep successfully before? 
One reason could be you are blocked by firewall at `3306` port. Can you connect to the database manually using -
`mysql --host=ensembldb.ensembl.org --user=anonymous` ?

Best regards,
Nakib

**Comment by nakib103** [comment]
Can you try using the mirror server listed here - https://www.ensembl.org/info/data/mysql.html, If any of them works for you.

You can change the database server  in VEP using the following options - https://www.ensembl.org/info/docs/tools/vep/script/vep_options.html#dbopt

**Comment by nakib103** [comment]
Hi, 

I am closing this issue now. But if you face any more problems please create a new issue or re-open this one and let us know.

Best regards,
Nakib
